docs/man/readtags.1.rst

```diff
@@ -72,6 +72,15 @@ Controlling the Output
 By default, the output of readtags contains only the name, input and pattern
 field. The Output can be tweaked using these options:
 
+``-C``, ``--canonicalize-input``
+	Reduct '..' and '.' in input fields of regular tags.
```

Comment:
I suggest using "resolve" rather than "reduce" because we are not just deleting `.` and `..`s.

Also, I think it's worth noting that the canonicalized path is unique (so we can compare them).

```suggestion
	Resolve '..' and '.' in input fields of regular tags.
	This produces a unique representation of the input path.
```